submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,150 @@
+
+As for the other fluids, you want to ensure that the water from your offshore pumps is reaching your heat exchangers without significant [throughput loss in pipelines](https://wiki.factorio.com/Fluid_system#Pipelines), hence make sure to have multiple pipelines along the way. Throughput loss can be observed when your heat exchangers don't have enough fluid input while the offshore pump output is below 1200/second. While the ideal approach is to simply have short pipelines, you can also consider using pumps. The critical number is a maximum of 17 pipe segments between every two pumps, in order to maintain the 1200 water/second from an offshore pump. Here it is worth noting that underground pipes pairs at any distance count as only two pipe segments. Secondly, you want to make sure the steam generated has enough pipe space to flow from exchangers to turbines without backing up, which is not an issue except for enormous nuclear plants. Generally, adding more pipes from heat exchangers to turbines is easier than adding pumps, as noted in [this post](https://www.reddit.com/r/factorio/comments/sbse5k/do_you_have_hard_time_estimating_pipeline/) by *u/warbaque* on Reddit.
+
+#### What about storing the steam?
+
+There is some debate in the community about whether steam storage is worthwhile. It is worth noting that it is not necessary for normal plant operation, since a correct ratio of exchangers to turbines (with *just* more than enough turbines) would ensure any steam produced is being consumed while the plant is running at full capacity. Meanwhile, adding optional components for steam storage, especially at a large scale, complicates your plant design. More importantly, it impacts the UPS by adding more fluid calculations. The size of this impact is arguably small compared to that of heat exchangers and steam turbines, based on my examinations and [these](https://www.reddit.com/r/technicalfactorio/comments/qc0npz/the_impact_of_nuclear_power_plants_on_ups/) [two](https://www.reddit.com/r/factorio/comments/s2h64o/100gw_nuclear_setup_ups_comparison_2x2_345ms_vs/) detailed explorations by technical community members (further discussed below). Meanwhile, steam storage offers some optional benefits but it is up to you to decide if you prefer these benefits at the cost of some extra space and UPS.
```

Comment:
```suggestion
There is some debate in the community about whether steam storage is worthwhile. It is worth noting that it is not necessary for normal plant operation, since a correct ratio of exchangers to turbines (with *just* more than enough turbines) would ensure any steam produced is being consumed while the plant is running at full capacity. Meanwhile, adding optional components for steam storage, especially at a large scale, complicates your plant design. More importantly, it impacts the UPS by adding more fluid calculations. The size of this impact is arguably small compared to that of heat exchangers and steam turbines, based on my examinations and [these](https://www.reddit.com/r/technicalfactorio/comments/qc0npz/the_impact_of_nuclear_power_plants_on_ups/) [two](https://www.reddit.com/r/factorio/comments/s2h64o/100gw_nuclear_setup_ups_comparison_2x2_345ms_vs/) detailed explorations by technical community members (further discussed below). Meanwhile, steam storage offers some optional benefits, but it is up to you to decide if you prefer these benefits at the cost of some extra space and UPS.
```